### DISCUSSION TITLE
Cancelling my subscription because GitHub Copilot requires bussiness subscription for self-signed certificates

### DISCUSSION THREAD

**Discussion by DuncanCasteleyn** [community/community#51126] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I have decided to cancel my GitHub Copilot subscription because self-signed certificates are only supported for Business subscriptions.

I'm very disappointed that I can't use GitHub Copilot because this hard block was implemented, the client I'm currently working for uses self-signed certificates in their internal network, I'm unable to use the product for the majority of my time coding.

I'd like this hard block to be removed, it's extremely unfriendly to individual users who decided they don't mind paying for it personally and forcing me to ask the client I'm working for to get GitHub Copilot Business subscription which they will reject anyway. I won't be taking a subscription again anytime soon as long as this limitation is still in place for personal users.

**Comment by tilmann1** [comment]
@DuncanCasteleyn Even as an individual user you can create a GitHub Team with yourself as the only member and then sign up for Copilot for Business to access this feature.

**Reply by DuncanCasteleyn** [reply]
> @DuncanCasteleyn Even as an individual user you can create a GitHub Team with yourself as the only member and then sign up for Copilot for Business to access this feature.

I'm not going to pay double for one simple feature, called allow self-signed certificates.

**Reply by jpop104** [reply]
Cancel my subscription!

**Comment by DuncanCasteleyn** [comment, marked as answer]
The solution given is pay for Copilot for Business which is exactly why I quit my subscription so this discussion simply ends with me not paying any longer for this service since it seems the team does not care to provide a solution for individual users.

If you want to use GitHub copilot behind something that needs self-signed certificates your only option will be to create a team and pay even more to use GitHub copilot, I don't find it okay you need pay extra for something as simple as that, so I refuse to continue my subscription.
